include/swift/AST/DiagnosticsSema.def

```diff
@@ -1063,6 +1063,9 @@ NOTE(unwrap_with_default_value,none,
 NOTE(unwrap_with_force_value,none,
      "force-unwrap using '!' to abort execution if the optional value contains "
      "'nil'", ())
+NOTE(optional_keypath_application_base, none,
```

Comment:
Could you move this below `optional_base_chain` so the keypath diagnostics stay together?

Reply:
Sure done! :)